databricks/koalas/indexes/datetimes.py

```diff
@@ -634,6 +636,49 @@ def strftime(self, date_format: str) -> Index:
         """
         return Index(self.to_series().dt.strftime(date_format))
 
+    def indexer_at_time(self, time: Union[datetime.time, str], asof: bool = False) -> Index:
```

Comment:
`asof` is not implemented in pandas yet, so there is not docstring for it.
https://github.com/pandas-dev/pandas/blob/v1.2.3/pandas/core/indexes/datetimes.py#L828-L865

Reply:
Shall we check it before we call `apply_batch`; otherwise the error will be messy.

Reply:
Good idea!